src/quequations.jl

```diff
@@ -1,61 +1,162 @@
 abstract QuEquation
 
+@doc """
+Schrodinger Equation type
+
+Fields :
+
+`hamiltonian` : Hamiltonian of the system is the only field for the type.
+""" ->
 immutable QuSchrodingerEq{H<:QuBase.AbstractQuMatrix} <: QuEquation
     hamiltonian::H
     QuSchrodingerEq(hamiltonian) = new(hamiltonian)
 end
 
+@doc """
+Schrodinger Equation method
+
+Input Parameters :
+
+`hamiltonian` : Hamiltonian of the system to construct `QuSchrodingerEq` type.
+""" ->
 QuSchrodingerEq{H<:QuBase.AbstractQuMatrix}(hamiltonian::H) = QuSchrodingerEq{H}(hamiltonian)
 
+@doc """
+Liouville von Neumann Equation type
+
+Fields :
+
+`liouvillian` : Liouvillian of the system is the only field for the type.
+""" ->
 immutable QuLiouvillevonNeumannEq{H<:QuBase.AbstractQuMatrix} <: QuEquation
     liouvillian::H
     QuLiouvillevonNeumannEq(liouvillian) = new(liouvillian)
 end
 
+@doc """
+Liouville von Neumann Equation method
+
+Input Parameters :
+
+`liouvillian` : Liouvillian of the system to construct `QuLiouvillevonNeumannEq` type.
+""" ->
 QuLiouvillevonNeumannEq{H<:QuBase.AbstractQuMatrix}(liouvillian::H) = QuLiouvillevonNeumannEq{H}(liouvillian)
 
-function liouvillian_op(h::QuBase.AbstractQuMatrix)
-    nb = size(coeffs(h), 1)
+@doc """
+Lindblad Master Equation type
+
+Fields :
+
+`lindblad` : Lindblad operator of the system
+""" ->
+immutable QuLindbladMasterEq{H<:QuBase.AbstractQuMatrix} <: QuEquation
+    lindblad::H
```

Comment:
It is better to keep the Hamiltionian and the collapse ops as well as we will need them for MCWF. The outer constructors should only accept Hamiltonian and a list of collapse ops (as before), so we can be sure that the cached Linblad matrix is consistent with those.